client/cache/bloom_filter_cache.go

```diff
@@ -0,0 +1,84 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/beego/beego/v2/core/berror"
+	"github.com/bits-and-blooms/bloom/v3"
+)
+
+type BloomFilterCache struct {
+	Cache
+	*bloom.BloomFilter
+	loadFunc       func(ctx context.Context, key string) (any, error)
+	expiration     time.Duration // set cache expiration, default never expire
+	notUpdateBloom bool          // update bloom key after put cache, default update
+}
+
+type BloomFilterCacheOption func(bfc *BloomFilterCache)
+
+func WithExpirationOpt(t time.Duration) BloomFilterCacheOption {
```

Comment:
Maybe we can add some prefix to avoid potential conflicts, i.e. `BFCWithExpiration()` ...
Expiration should be a common configure for other cache implementation.